BedrockCore.cpp

```diff
@@ -15,9 +15,11 @@ bool BedrockCore::peekCommand(BedrockCommand& command) {
     STable& content = command.jsonContent;
     SDEBUG("Peeking at '" << request.methodLine << "'");
     command.peekCount++;
+    uint64_t timeout = command.request.isSet("timeout") ? command.request.calc("timeout") : DEFAULT_TIMEOUT;
```

Comment:
`"timeout"` at first made me think it's the command timeout... but it isn't, it really is "queryTimeout" or something like that, correct? If so, I feel like we should rename the parameter/variables to match what this currently vague "timeout" is (vague until you read all of the implementation, of course)

Reply:
It's intended as a command timeout, and works as such unless you do something highly unusual. I think it has the right name.

Reply:
So I misunderstood? This timeout is _not_ only for queries? eg if the command does a `sleep(61seconds)`;, is it going to complete or not?

Reply:
It is only checked during queries. This will complete:

```
SELECT 1;
sleep(61);
```

This will not: 

```
sleep(61);
SELECT 1;
```

The first one times out because the timeout is checked at the end of the query, and it's fine, but then we do other, non-query activity for 61 seconds, and do not check the timeout again.

The second one does not complete, because we run past the timeout, and then do a query, which will show 61 seconds elapsed, and it will timeout.

similarly, this will timeout:
```
while (1) {
    SELECT 1;
}
```
because eventually, at the end of one of the queries, the timeout will have elapsed.

Being that all of our real-world commands do a lot of queries but not a lot of sleeping, the timeout works how you'd think for all our real-world cases.

Reply:
Ohhhh! Indeed, I had misunderstood! Thanks for the explanation :) Perhaps worth documenting somewhere